pkg/sources/github/github.go

```diff
@@ -574,7 +575,7 @@ func (s *Source) enumerateWithApp(ctx context.Context, apiEndpoint string, app *
 	// This client is required to create installation tokens for cloning.
 	// Otherwise, the required JWT is not in the request for the token :/
 	appItr, err := ghinstallation.NewAppsTransport(
```

Comment:
Instead of storing the original transport in a `originalTransport`, can we just move this ahead of where `s.httpClient.Transport` is modified?

In either case, a comment of why this var or order of things is necessary could be useful in the future.